api/webhook/webhook.go

```diff
@@ -0,0 +1,141 @@
+package webhook
+
+import (
+	"context"
+
+	"github.com/caraml-dev/merlin/log"
+	"github.com/caraml-dev/merlin/models"
+	"github.com/caraml-dev/mlp/api/pkg/webhooks"
+	webhookManager "github.com/caraml-dev/mlp/api/pkg/webhooks"
+)
+
+var (
+	OnModelCreated = webhooks.EventType("on-model-created")
```

Comment:
if we have a dedicated function for each event do we need to expose this?

Reply:
made changes for this! there will only be one function in interface which will be used by all event. please recheck it, thank you